src/AutoRest.CSharp/LowLevel/Generation/LowLevelClientWriter.cs

```diff
@@ -72,7 +72,7 @@ private void WriteClientMethodBody(CodeWriter writer, LowLevelClientMethod clien
             {
                 writer.Line($"options ??= new {typeof(Azure.RequestOptions)}();");
 
-                writer.Append($"{typeof(Azure.Core.HttpMessage)} message = {RequestWriterHelpers.CreateRequestMethodName(clientMethod.Name)}(");
```

Comment:
While you're at it, can you please add `CodeWriterDeclaration` for the message variable?

Reply:
Added!